### PR TITLE
Tyrrrz.LightBulb version 2.3.3

### PR DESCRIPTION
**Pull request by KaranKad**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17742)

### CODE REVIEW

File: manifests/t/Tyrrrz/LightBulb/2.3.3/Tyrrrz.LightBulb.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+PackageIdentifier: Tyrrrz.LightBulb
+PackageVersion: 2.3.3
+InstallerType: exe
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/Tyrrrz/LightBulb/releases/download/2.3.3/LightBulb-Installer.exe
+  InstallerSha256: C47F4E2945ED2B743EBB37ACFE517A33F256D358BAF362CF835F52E29D5DF609
+  InstallerSwitches:
+    Silent: /VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-
+    SilentWithProgress: /VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-
```
Comment (OfficialEsco) [inline manifests/t/Tyrrrz/LightBulb/2.3.3/Tyrrrz.LightBulb.installer.yaml:13]:
```suggestion
  InstallerSwitches:
    Custom: /NORESTART
```
Just a bit of clean up

File: manifests/t/Tyrrrz/LightBulb/2.3.3/Tyrrrz.LightBulb.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Tyrrrz.LightBulb
+PackageVersion: 2.3.3
+PackageLocale: en-US
+Publisher: Tyrrrz
```
Comment (OfficialEsco) [inline manifests/t/Tyrrrz/LightBulb/2.3.3/Tyrrrz.LightBulb.locale.en-US.yaml:7]:
```suggestion
Publisher: Alexey 'Tyrrrz' Golub
```

File: manifests/t/Tyrrrz/LightBulb/2.3.3/Tyrrrz.LightBulb.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+PackageIdentifier: Tyrrrz.LightBulb
+PackageVersion: 2.3.3
+PackageLocale: en-US
+Publisher: Tyrrrz
+Author: Alexey Golub
+PackageName: LightBulb
+PackageUrl: https://github.com/Tyrrrz/LightBulb
+License: Copyright (C) 2017-2020 Alexey Golub; GNU General Public License
+LicenseUrl: https://github.com/Tyrrrz/LightBulb/blob/master/License.txt
```
Comment (OfficialEsco) [inline manifests/t/Tyrrrz/LightBulb/2.3.3/Tyrrrz.LightBulb.locale.en-US.yaml:12]:
```suggestion
License: GNU General Public License Version 3
LicenseUrl: https://github.com/Tyrrrz/LightBulb/blob/master/License.txt
Copyright: Copyright (C) 2017-2021 Alexey Golub
CopyrightUrl: https://github.com/Tyrrrz/LightBulb/blob/master/License.txt
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Cleanup + ARP matching
We could also add `ProductCode: {892F745F-A497-42ED-B503-8D74936D0BEB}_is1` i think, but not sure if it changes so i didnt.

**Comment by OfficialEsco** [comment]
Uhoh, lemme check

File: manifests/t/Tyrrrz/LightBulb/2.3.3/Tyrrrz.LightBulb.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Tyrrrz.LightBulb
+PackageVersion: 2.3.3
+InstallerType: exe
```
Comment (OfficialEsco) [inline manifests/t/Tyrrrz/LightBulb/2.3.3/Tyrrrz.LightBulb.installer.yaml:6]:
```suggestion
InstallerType: inno
```
🤦‍♂️ didn't see that one

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
🤦‍♂️

**Review by OfficialEsco — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**